game/CMakeLists.txt

```diff
@@ -99,6 +99,17 @@ target_link_libraries(performous PRIVATE ${LibEpoxy_LIBRARIES})
 find_package(PkgConfig REQUIRED)
 pkg_check_modules(deps REQUIRED IMPORTED_TARGET glib-2.0 gio-2.0 gobject-2.0)
 
+option(ALLOW_SELF_BUILT_AUBIO "allow downloading and building aubio from sources if not available on system" FALSE)
+option(FORCE_SELF_BUILT_AUBIO "force downloading and building aubio from sources" FALSE)
+find_package(Aubio 0.4.9 REQUIRED)
+message(STATUS "Aubio includes: ${Aubio_INCLUDE_DIRS}")
+target_include_directories(performous SYSTEM PRIVATE ${Aubio_INCLUDE_DIRS})
+target_link_libraries(performous PRIVATE ${Aubio_LIBRARIES})
+add_definitions(${Aubio_DEFINITIONS})
+
+option(ALLOW_SELF_BUILT_JSON "allow downloading and building nlohmann_json from sources if not available on system" FALSE)
+option(FORCE_SELF_BUILT_JSON "force downloading and building nlohmann_json from sources" FALSE)
```

Comment:
Thise 2 boolean seems to conflict each other, what would mean ALLOW=FALSE and FORCE=TRUE? Maybe would be better to use a single variable wit 3 values (SEFL_BUILD_XXX={NEVER,AUTO,ALWAYS}?

Reply:
I'd agree with @OznOg here, 1 variable with 3 values